services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/TableIndexManagerImpl.java

```diff
@@ -82,7 +82,7 @@ public class TableIndexManagerImpl implements TableIndexManager {
 
 	public static final int MAX_MYSQL_INDEX_COUNT = 60; // mysql only supports a max of 64 secondary indices per table.
 
-	public static final long MAX_BYTES_PER_BATCH = 1024*1024*5;// 5MB
+	public static final long MAX_BYTES_PER_BATCH = 1024*1024*10;// 10MB

```

Comment:
This is more of a concern for what the worker machine memory can handle rather than what the DB can handle.  If we allow 20+ threads on one machine to build views at the same time then we might need 2 GB of the machines 8 GB.